src/Daemon.hs

```diff
@@ -125,10 +143,27 @@ timeoutDaemon' logger name delay computation =
 
 -- * Log Daemon
 
--- | Create a log daemon
+-- | Create a log daemon.  This is both responsible for the moderator's event log (file location
+-- given in Config.hs) and system log messages (stderr).
+--
+-- NOTE: 'logDaemon' can only be called once, and only have one thread.  Multiple calls to 'start'
+-- will return the same 'ThreadId', multiple calls to 'logDaemon' will crash.  (What would go wrong?
+-- If too many `logDaemon`s are running concurrently, they might undo each others moderator event
+-- log entries, or enter an inconsistent state.  also, stderr could be garbled if two lines are
+-- printed concurrently.)
 logDaemon :: LogConfig -> IO (MsgDaemon LogEntry)
```

Comment:
This could be resolved in the future separating moderator logs from system logs with two different types and log daemons.